internal/graph/dotgraph.go

```diff
@@ -472,3 +473,8 @@ func min64(a, b int64) int64 {
 	}
 	return b
 }
+
+func escapeDoubleQuotes(s string) string {
+	re := regexp.MustCompile(`\\([\s\S])|(")`)
```

Comment:
The regular expression should be compiled once, not on every function call.

Reply:
Ack. The regex is no longer used.